QueueTest/UnitTest1.cs

```diff
@@ -0,0 +1,51 @@
+using Training;
+namespace QueueTest {
+   [TestClass]
+   public class UnitTest1 {
+      TQueue<int> mNumbers = new ();
+      Queue<int> mQueueNum = new ();
+
+      [TestMethod]
+      public void EnqueueTest () {
+         mNumbers.Enqueue (1);

```

Comment:
Instead of repeating this have a list or array that holds the numbers 1, 2, 3 and then in a loop add them into both mTQueue and mQueue.
Also do Enqueue and Dequeue randomly like 3 enqueus 2 deques 4 enques like that